docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -120,20 +120,23 @@ lockScreenConfig.showDeviceLogo(false);
 !@
 
 ## Creating a Custom Lock Screen
-If you would like to use your own lock screen instead of the one provided by the library, but still use the logic we provide, you can use a new initializer within @![iOS]`SDLLockScreenConfiguration`!@ @![android]`LockScreenConfig`!@:
+If you would like to use your own lock screen instead of the one provided by the library, but still use the logic we provide, you can use a new initializer within @![iOS]`SDLLockScreenConfiguration`. As of iOS 13, presented `UIViewController`s are now dismissible by swiping down on the phone screen. Unless the OEM has enabled Passenger Mode, lock screens should not be dismissible by the user. To prevent this, set the `modalPresentationStyle` property of your custom lock screen `UIViewController` to `full screen`. Not doing so may result in your app being rejected by OEMs.!@ @![android]`LockScreenConfig`:!@
```

Comment:
```suggestion
If you would like to use your own lock screen instead of the one provided by the library, but still use the logic we provide, you can use a new initializer within @![iOS]`SDLLockScreenConfiguration`. As of iOS 13, presented `UIViewController`s are now dismissible by swiping down on the phone screen. Unless the OEM has enabled Passenger Mode, lock screens should not be dismissible by the user. To prevent this, set the `modalPresentationStyle` property of your custom lock screen `UIViewController` to `fullScreen`. Not doing so may result in your app being rejected by OEMs.!@ @![android]`LockScreenConfig`:!@
```

Reply:
@![android]`LockScreenConfig`:!@  

got separated from its iOS counterpart: 

 @![iOS]`SDLLockScreenConfiguration`

Just get rid of the colon after `LockScreenConfig` and replace it with a period.

Reply:
Addressed in most recent commit.